### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by yangfl**

### CODE REVIEW

**Comment by def-** [comment]
Thanks. Did you use scspell? Has your tool created a dictionary / index so that we can keep using and updating it in the future?

File: src/base/system.h [hunk 1]
```diff
@@ -189,7 +189,7 @@ IOHANDLE io_open(const char *filename, int flags);
 
 	Parameters:
 		io - Handle to the file to read data from.
-		buffer - Pointer to the buffer that will recive the data.
+		buffer - Pointer to the buffer that will receive the data.
```
Comment (heinrich5991) [inline src/base/system.h]: Oh god, this one bugged me for so long, but I didn't bother fixing it.

**Comment by yangfl** [comment]
Nope, I use codespell which does not generate any index files, and it does make many false positives. You could use your own spell checker, of course.

**Comment by def-** [comment]
No no, that's fine. I just remembered that I ran scspell on some other code base and then it would remember all the results and next time you only have to judge the changes.

**Pull request merged**